sonic-pcied/scripts/pcied

```diff
@@ -84,6 +128,27 @@ class DaemonPcied(DaemonBase):
             self.update_state_db("PCIE_DEVICES", "status", "PASSED")
             self.log_info("PCIe device status check : PASSED")
 
+        # update AER-attributes to DB
+        for item in resultInfo:
+            if item["result"] == "Failed":
+                continue
+
+            Bus = item["bus"]
+            Dev = item["dev"]
+            Fn = item["fn"]
+            Id = item["id"]
+
+            # construct sysfs device path
+            device_path = "/sys/bus/pci/devices/0000:%s:%s.%s" % (Bus, Dev, Fn)
+            device = os.path.join(device_path, "device")
+
+            if not os.path.isfile(device):
+                continue
+
+            device_name = "0x%s|%s:%s.%s" % (Id, Bus, Dev, Fn)
+            pcie_dev_key = "PCIE_DEVICE|" + device_name
+            self.update_aer_to_statedb(device_path, device_name, pcie_dev_key)
+
```

Comment:
This only needs to be done for the PCIE_DEVICES status as FAILED.  This can be performed at line#122. Can this be done by a separate api too?